pocs/utils/images/__init__.py

```diff
@@ -6,7 +6,10 @@
 from matplotlib import pyplot as plt
 from warnings import warn
 
+from astropy.io import fits
```

Comment:
I don't think you are using this.

Reply:
Nope, using that in my jupyter notebook.